dnf-behave-tests/dnf/plugins-core/offline-upgrade-security-filters.feature

```diff
@@ -0,0 +1,340 @@
+       Package         Architecture    Version        Repository                 Size
+      ================================================================================
+      Upgrading:
+       B               x86_64          2-2            security-upgrade          6.0 k
+       dracut          x86_64          2-2            security-upgrade          6.0 k
+      
+      Transaction Summary
+      ================================================================================
+      Upgrade  2 Packages
+      """
+ When I execute dnf with args "offline-upgrade download --security"
+ Then the exit code is 0
+  And stdout contains lines
+      """
+      Dependencies resolved.
+      ================================================================================
+       Package     Architecture     Version          Repository                  Size
+      ================================================================================
+      Upgrading:
+       B           x86_64           2-2              security-upgrade           6.0 k
+      
+      Transaction Summary
+      ================================================================================
+      Upgrade  1 Package
+      """
+
+
+@bz1939975
+Scenario: Test security severity filter with offline-upgrade
+Given I use repository "dnf-ci-security"
+  And I execute dnf with args "install bugfix_B-1.0-1 advisory_B-1.0-3"
+ Then the exit code is 0
+ When I execute dnf with args "offline-upgrade download"
+ Then the exit code is 0
+  And stdout contains lines
+      """
+      Dependencies resolved.
+      ================================================================================
+       Package            Architecture   Version        Repository               Size
+      ================================================================================
+      Upgrading:
+       advisory_B         x86_64         1.0-4          dnf-ci-security         6.0 k
+       bugfix_B           x86_64         1.0-2          dnf-ci-security         6.0 k
+      
+      Transaction Summary
+      ================================================================================
+      Upgrade  2 Packages
+      """
+ When I execute dnf with args "offline-upgrade download --secseverity=Critical"
+ Then the exit code is 0
+  And stdout contains lines
+      """
+      Dependencies resolved.
+      ================================================================================
+       Package            Architecture   Version        Repository               Size
+      ================================================================================
+      Upgrading:
+       advisory_B         x86_64         1.0-4          dnf-ci-security         6.0 k
+      
+      Transaction Summary
+      ================================================================================
+      Upgrade  1 Package
+      """
```

Comment:
Can you add a package with a lower-severity security update to also test that only "Critical" security updates are applied?